### PR TITLE
workers-checker improvements

### PR DESCRIPTION
**Pull request by OriHoch**
CloudWebManage/cwm-worker-cluster#199

* add support for parallelization
* use cwm-worker-deployment.get_health to get the minio health details
* keep last 100 health statuses in local storage
* use multiple methods to get namespace names: helm releases, get namespaces from k8s, from worker health keys
* for namespaces which get_health return None (meaning namespace doesn't exist) - delete the worker_health key

### CODE REVIEW

File: cwm_worker_operator/deployments_manager.py [hunk 1]
```diff
@@ -168,6 +168,14 @@ def deploy(self, deployment_config, **kwargs):
     def is_ready(self, namespace_name, deployment_type, minimal_check=False):
         return cwm_worker_deployment.deployment.is_ready(namespace_name, deployment_type, minimal_check=minimal_check)
 
+    def get_health(self, namespace_name, deployment_type):
+        return cwm_worker_deployment.deployment.get_health(namespace_name, deployment_type)
+
+    def get_worker_id_namespaces(self):
+        ret, out = subprocess.getstatusoutput("kubectl get ns | tail -n+2 | cut -d' ' -f1")
```
Comment (iamazeem) [inline cwm_worker_operator/deployments_manager.py:175]: Suggested change for command:

```bash
kubectl get ns --no-headers -o=custom-columns=NAME:.metadata.name
```

i.e.

```python
ret, out = subprocess.getstatusoutput('kubectl get ns --no-headers -o=custom-columns=NAME:.metadata.name')
```

File: cwm_worker_operator/config.py [hunk 2]
```diff
@@ -135,3 +135,4 @@
 REDIS_CLEANER_DELETE_ANY_HOSTNAME_ERROR_MIN_SECONDS = int(os.environ.get('REDIS_CLEANER_DELETE_ANY_HOSTNAME_ERROR_MIN_SECONDS') or '7200')
 
 WORKERS_CHECKER_SLEEP_TIME_BETWEEN_ITERATIONS_SECONDS = int(os.environ.get('WORKERS_CHECKER_SLEEP_TIME_BETWEEN_ITERATIONS_SECONDS') or '5')
+WORKERS_CHECKER_MAX_PARALLEL_DEPLOY_PROCESSES = int(os.environ.get('WORKERS_CHECKER_MAX_PARALLEL_DEPLOY_PROCESSES') or 2)
```
Comment (iamazeem) [inline cwm_worker_operator/config.py:138]: Is `2` without quotes intentional?

**Review by iamazeem — APPROVED** [review]
Looks good! :+1:

File: cwm_worker_operator/deployments_manager.py (hunk 1 above)
Comment (OriHoch) [reply cwm_worker_operator/deployments_manager.py]: thanks, changed

File: cwm_worker_operator/config.py (hunk 2 above)
Comment (OriHoch) [reply cwm_worker_operator/config.py]: it works that way too, but I changed to string to be the same as other similar configurations

**Pull request merged**